adback-solution-to-adblock/includes/class-ad-back-updator.php

```diff
@@ -136,6 +136,25 @@ public static function isRewriteRouteEnabled()
         return (bool) get_option('permalink_structure');
     }
 
+    public static function isRewriteRulesConflictWithEndpoints($endpoint)
+    {
+        require_once( ABSPATH . 'wp-includes/class-wp-rewrite.php' );
```

Comment:
why do you need to require the class `WP_Rewrite` ?

Reply:
I'm not sure this has to be done like this.

Have a loot at WP_rewrite usage: https://codex.wordpress.org/Class_Reference/WP_Rewrite

```global $wp_rewrite;```  
```die(var_dump($wp_rewrite->rules)));```

Reply:
Yes I don't need this require anymore with `get_option()` function